### PR TITLE
Update SUSEConnect keep-alive docs

### PR DESCRIPTION
**Pull request by lagartoflojo**
Mention the benefits of this new feature at the beginning of the article.

### CODE REVIEW

File: suseconnect-tracking/xml/art_suseconnect_tracking.xml [hunk 1]
```diff
@@ -56,9 +56,14 @@
   </tip>
 
   <para>
-   With newer versions of &suseconnect;, each registered host contacts SCC or RMT automatically on a
-   daily basis. It sends the unique identification of the host and description of 
-   its hardware environment.
+   SUSE is committed to helping provide better insights into the consumption of SUSE subscriptions,
```
Comment (tbazant) [inline suseconnect-tracking/xml/art_suseconnect_tracking.xml:59]:
```suggestion
   &suse; is committed to helping provide better insights into the consumption of SUSE subscriptions,
```

**Review by tbazant — APPROVED** [review]
just a nitpick

**Comment by tbazant** [comment]
Adding @dariavladykina  for a language review, then we can merge it

File: suseconnect-tracking/xml/art_suseconnect_tracking.xml [hunk 2]
```diff
@@ -56,9 +56,14 @@
...
 
   <para>
-   With newer versions of &suseconnect;, each registered host contacts SCC or RMT automatically on a
-   daily basis. It sends the unique identification of the host and description of 
-   its hardware environment.
+   &suse; is committed to helping provide better insights into the consumption of SUSE subscriptions,
+   regardless of where they are running or how they are managed; physical or virtual, on-prem or in
+   the cloud, connected to SCC or RMT, or managed by SUSE Manager.
+
+   To help you identify or filter out systems in SCC that are no longer running or decommissioned,
```
Comment (dariavladykina) [inline suseconnect-tracking/xml/art_suseconnect_tracking.xml:63]:
```suggestion
   To help you identify or filter out systems in SCC that are decommissioned or no longer running,
```

File: suseconnect-tracking/xml/art_suseconnect_tracking.xml [hunk 3]
```diff
@@ -56,9 +56,14 @@
...
+   &suseconnect; now features a daily “ping,” which will update system information automatically.
```
Comment (dariavladykina) [inline suseconnect-tracking/xml/art_suseconnect_tracking.xml:64]:
```suggestion
   &suseconnect; now features a daily <quote>ping,</quote> which will update system information automatically.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see minor nits.

File: suseconnect-tracking/xml/art_suseconnect_tracking.xml [hunk 4]
```diff
@@ -56,9 +56,14 @@
...
   <para>
-   With newer versions of &suseconnect;, each registered host contacts SCC or RMT automatically on a
-   daily basis. It sends the unique identification of the host and description of 
-   its hardware environment.
+   &suse; is committed to helping provide better insights into the consumption of SUSE subscriptions,
+   regardless of where they are running or how they are managed; physical or virtual, on-prem or in
+   the cloud, connected to SCC or RMT, or managed by SUSE Manager.
+
+   To help you identify or filter out systems in SCC that are decommissioned or no longer running,
+   &suseconnect; now features a daily <quote>ping,</quote> which will update system information automatically.
```
Comment (tbazant) [inline suseconnect-tracking/xml/art_suseconnect_tracking.xml:64]:
```suggestion
   &suseconnect; now features a daily 'ping', which will update system information automatically.
```

File: suseconnect-tracking/xml/art_suseconnect_tracking.xml [hunk 5]
```diff
@@ -56,9 +56,14 @@
...
   <para>
-   With newer versions of &suseconnect;, each registered host contacts SCC or RMT automatically on a
-   daily basis. It sends the unique identification of the host and description of 
-   its hardware environment.
+   &suse; is committed to helping provide better insights into the consumption of SUSE subscriptions,
+   regardless of where they are running or how they are managed; physical or virtual, on-prem or in
+   the cloud, connected to SCC or RMT, or managed by SUSE Manager.
+
+   To help you identify or filter out systems in SCC that are decommissioned or no longer running,
+   &suseconnect; now features a daily 'ping', which will update system information automatically.
```
Comment (dariavladykina) [inline suseconnect-tracking/xml/art_suseconnect_tracking.xml:64]:
```suggestion
   &suseconnect; now features a daily 'ping,' which will update system information automatically.
```

**Review by dariavladykina — COMMENTED** [review]
Just one more nit.

File: suseconnect-tracking/xml/art_suseconnect_tracking.xml [hunk 6]
```diff
@@ -56,9 +56,14 @@
...
   <para>
-   With newer versions of &suseconnect;, each registered host contacts SCC or RMT automatically on a
-   daily basis. It sends the unique identification of the host and description of 
-   its hardware environment.
+   &suse; is committed to helping provide better insights into the consumption of SUSE subscriptions,
+   regardless of where they are running or how they are managed; physical or virtual, on-prem or in
+   the cloud, connected to SCC or RMT, or managed by SUSE Manager.
+
+   To help you identify or filter out systems in SCC that are decommissioned or no longer running,
+   &suseconnect; now features a daily 'ping,' which will update system information automatically.
```
Comment (tbazant) [inline suseconnect-tracking/xml/art_suseconnect_tracking.xml:64]:
```suggestion
   &suseconnect; now features a daily 'ping', which will update system information automatically.
```

**Review by tbazant — APPROVED** [review]
LG

**Pull request merged**